src/components/graph-line/graph-line.css

```diff
@@ -1,7 +1,7 @@
 .graph-line {
   display: flex;
   width: 100%;
-  height: 34px;
```

Comment:
why this

Reply:
Because this element should have height of 35px.